src/Components/BookDetail/Reviews/ReviewSummary/ReviewSummary.jsx

```diff
@@ -0,0 +1,27 @@
+function ReviewSummary ({reviews}) {
+
+    let reviewCount = {reviews}.reviews.length;
```

Comment:
Please remove semi-colons at the ends of lines 3,4,5,6.  If these are necessary for the code to work, please ignore

Reply:
Done, semi-colons removed.